#!/usr/bin/python3


class CustomError(Exception):
    pass
